elastic/kibana
# [Uptime] show monitor management button on-prem

## Summary

Relates to https://github.com/elastic/uptime/issues/475

Shows monitor management button when users are not on Cloud.

Testing
--
1. Change the `renderApp` function in Uptime public plugin to `renderApp(coreStart, plugins, corePlugins, params, false)`
2. Ensure that the monitor management button appears in the Uptime header
